### ISSUE TITLE
[Accessibility- Tab order C 12.8] News_details- Ensure consistent tab order

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3023]
**Environment:** Tribe

**Findings:**
In article, we don't have a consistent tabbing path through content and functionality:

- Tabbing goes to additional actions before favorites => the order must be reversed

- Tabbing does not allow access to the “See more” button of like reactions

- Tabbing does not allow access to the functionalities of modal windows (drawer) by clicking directly on the buttons that allow them to open:

- [ ] The drawer to display all comments

- [ ] The drawer to display all like reactions

- [ ] The drawer to send a kudos or Share an article

- [ ] The Reference in notes drawer

**Expected behavior:**

- Tabbing shouldn't reach visually hidden elements.

- When activating a button that opens a modal window, the focus should be moved to the first element of the window, and when closing it, the focus should return to the activation button
